services/get-message.yml

```diff
@@ -17,11 +17,11 @@
   result: the_message
 
 - step: assign
-  type: assign
+  call: assign
   assign:
     stringValue: "Bürokratt"
     integerValue: 2021
 
 - step: return_value
-  type: return
-  returnValue: ${the_message.body}
+  call: return
```

Comment:
Remove `call: return` and just use `return: ${the_message.body}`. It means that `return` itself is a key acting as a function.